downloaded/installLisk.sh

```diff
@@ -239,7 +239,12 @@ upgrade_lisk() {
 
 	echo "Copying config.json entries from previous installation"
 	OLD_VERSION=$( "$LISK_INSTALL/bin/jq" --raw-output .version "$LISK_BACKUP/package.json" )
-	"$LISK_INSTALL/bin/node" "$LISK_INSTALL/scripts/update_config.js" --network "${RELEASE}net" --output "$LISK_INSTALL/config.json" "$LISK_BACKUP/config.json" "$OLD_VERSION"
+	if [ -f "$LISK_INSTALL/framework/src/modules/chain/scripts/update_config.js" ]; then
```

Comment:
this change depends on https://github.com/LiskHQ/lisk/issues/3255, once the core issue is fixed we can adjust the path accordingly